code/modules/economy/cael/ATM.dm

```diff
@@ -326,15 +326,18 @@
 			if("e_withdrawal")
 				var/amount = max(text2num(href_list["funds_amount"]),0)
 				amount = round(amount, 0.01)
+				var/obj/item/charge_stick/E
 				if(amount <= 0)
 					alert("That is not a valid amount.")
+				else if(amount > initial(E.max_worth))
+					alert("That amount exceeds the max amount holdable by basic charge sticks.")
 				else if(authenticated_account && amount > 0)
 					//create an entry in the account transaction log
 					if(authenticated_account.withdraw(amount, "Credit withdrawal", machine_id))
 						playsound(src, 'sound/machines/chime.ogg', 50, 1)
-						var/obj/item/charge_card/E = new(loc)
+						E = new(loc)
```

Comment:
Not required for this PR, but as a note, maybe it would be worth making credsticks a printed or traded object (maybe bought from the banking provider) rather than produced via magic - would make having your credstick stolen a bit more annoying too.